### PR TITLE
Add python code examples

### PR DESCRIPTION
**Pull request by sachaarbonel**

### CODE REVIEW

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 1]
```diff
@@ -98,8 +101,22 @@ function helloWorld() {
...
+        "created_by_id": "user_id",
+        "settings_override": {"audio": {"access_request_enabled": False}},
+    },
+    "members": [{"role": "speaker", "user_id": "user_id"}],
+}
+
+call_id = "your_call_id"
+
+client = Stream(api_key="your_api_key", api_secret="your_api_secret")
+response = client.video.get_or_create_call(calltype=calltype_name, callid=call_id, data=call_request_data)
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:118]: can we go with `call_type` and `call_id` ?

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 2]
```diff
@@ -208,6 +225,43 @@ func main() {
...
+
+```py
+from getstream import Stream
+
+call_id = "your_call_id"
+calltype = "default"
+
+client = Stream(api_key="your_api_key", api_secret="your_api_secret")
+response = client.video.start_broadcasting(callid=call_id, calltype=calltype)
+print(response)
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:249]: probably good to remove prints

File: docusaurus/video/docusaurus/shared [hunk 3]
```diff
@@ -0,0 +1 @@
+/Users/sachaarbonel/.nvm/versions/node/v16.14.2/bin/../lib/node_modules/stream-chat-docusaurus-cli/shared
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/shared:1]: add to .gitignore?

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 4]
```diff
@@ -51,8 +51,11 @@ function helloWorld() {
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+from getstream import Stream
+
+client = Stream(api_key="your_api_key", api_secret="your_api_secret")
+token = client.create_token()
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:57]: aren't you missing the user here?

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 5]
```diff
@@ -142,8 +151,21 @@ call.getOrCreate({
...
+        "created_by_id": "user_id",
+        "settings_override": {"audio": {"access_request_enabled": False}},
+    },
+    "members": [{"role": "speaker", "user_id": "user_id"}],
+}
+
+call_id = "your_call_id"
+
+client = Stream(api_key="your_api_key", api_secret="your_api_secret")
+response = client.video.get_or_create_call(call_type=calltype_name, call_id=call_id, data=call_request_data)
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:168]: @sachaarbonel it is probably nicer to have a `Call` class that captures the type/id params

```python
call = client.video.call("default", "id")
call.create()
call.start_broadcasting()
...
```

Comment (sachaarbonel) [reply docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:168]: it's done on the sdk side, updating the docs now

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 6]
```diff
@@ -222,8 +245,15 @@ call.updateCallMembers({
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+call = client.video.call("default", "id")
+call.update_call_members(
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:249]: maybe nicer to unpack the request type a bit
```python
def update_call_members(members):
    pass
```

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 7]
```diff
@@ -303,8 +333,16 @@ call.update({ custom: { color: "red" } });
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+call = client.video.call("default", "id")
+call.update(
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:337]: same bubbling

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 8]
```diff
@@ -391,8 +430,16 @@ call.stopBroadcasting();
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+from getstream import Stream
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:433]: no need to import and instantiate everything

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 9]
```diff
@@ -464,8 +511,13 @@ const address = `${resp.call.ingress.rtmp.address}/<callType>/<callID>/<apiKey>/
...
 ```py
-def hello_world():
-    print("Hello, world!")
+call_type = "your_call_type"
+call_id = "your_call_id"
+data = {}  # Add any data you need for the get_or_create_call method
+call = client.video.call(call_type, call_id)
+response = call.create(data=data)
+rtmp_address = response["call"]["ingress"]["rtmp"]["address"]
+address = f"{rtmp_address}/{call_type}/{call_id}/{VIDEO_API_KEY}/{client.create_token()}"
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:520]: > response["call"]["ingress"]["rtmp"]["address"] 

this should be the fully qualified URL with api key and call id info included (rtmps://stream.io/call/default/123?apikey)

the stream key should be created using client.create_token() (user_id needs to be there)

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 10]
```diff
@@ -567,8 +621,10 @@ call.queryRecordings("<session ID>");
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+call_type="your_call_type"
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:624]: inline vars as args

Comment (tbarbugli) [reply docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:624]: > client.video.call("default", "123")

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx (hunk 9 above)
Comment (tbarbugli) [reply docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:520]: use "livestream" call type ;)

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 11]
```diff
@@ -567,8 +596,7 @@ call.queryRecordings("<session ID>");
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+recordings = call.query_recordings(sessionid="your_session_id")
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:599]: let's pick a better example, in most cases you dont know what the session_id is (missing an _ there btw)
I think a better example for this is to show first page and getting the next page

Comment (sachaarbonel) [reply docusaurus/video/docusaurus/docs/api/basics/get_started.mdx]: We currently don't do pagination
Capture d’écran 2023-07-12 à 13 00 46

File: docusaurus/video/docusaurus/docs/api/basics/get_started.mdx [hunk 12]
```diff
@@ -464,8 +488,11 @@ const address = `${resp.call.ingress.rtmp.address}/<callType>/<callID>/<apiKey>/
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+data = {}  # Add any data you need for the get_or_create_call method
+call = client.video.call("livestream", "id")
+response = call.create(data=data,members=[{user_id: "sacha"}])
```
Comment (tbarbugli) [inline docusaurus/video/docusaurus/docs/api/basics/get_started.mdx:493]: I dont think this works if you create a call (there is no created_by field)

**Pull request closed without merging**